godotproject/Level0AttractMode.gd

```diff
@@ -0,0 +1,50 @@
+extends Node2D
+
+# Each level has to have a name in it now, this will be displayed on the high score screen
+const level_name := "Morning Meadow"
```

Comment:
not used in this implementation

Reply:
I'm gonna leave it here since it shouldn't affect anything and I don't want to somehow break it by taking it out. No harm done by it being here